packages/blade/src/components/Accordion/_decisions/decisions.md

```diff
@@ -63,12 +125,30 @@ import { Accordion, AccordionItem } from '@razorpay/blade';
 
 ### AccordionItem
 
+| Key                         | Type                            | Default      | Description                                                                                                                                       | Required |
+| --------------------------- | ------------------------------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| title (⚠️ deprecated)       | `string`                        | `undefined`  | Title text content                                                                                                                                |          |
+| description (⚠️ deprecated) | `string`                        | `undefined`  | Body text content                                                                                                                                 |          |
+| icon (⚠️ deprecated)        | `IconComponent`                 | `undefined`  | Renders a Blade icon as title prefix (requires `showNumberPrefix={false}`)                                                                        |          |
+| variant                     | `bordered`, `borderless`        | `borderless` | Visual variant of AccordionItem                                                                                                                   |          |
+| children                    | `JSX`                           | `undefined`  | Accepts [`AccordionItemTrigger`](#accordionitemtrigger) and [`AccordionItemBody`](#accordionitembody). Accepting other JSX elements is deprecated | ✅       |
+| isExpanded                  | `boolean`                       | `undefined`  | Expanded state control when individual item is used                                                                                               |          |
+| onExpandChange              | (isExpanded: `boolean`) => void | `undefined`  | Callback for handling change in expand state when individual item is used                                                                         |          |
```

Comment:
Do we need these props? I think we should not allow individual usage of `AccordionItem`, it should always be wrapped within `Accordion`.

```js
<Accordion defaultExpandedIndex="0" onExpandChange={() => {}} >
 <AccordionItem>
    <AccordionItemTrigger
      title="Accordion Title"
      icon={QRIcon}
    />
    <AccordionItemBody>
      <Text>Information inside Accordion</Text>
    </AccordionItemBody>
  </AccordionItem>
 </Accordion>
```

Reply:
yes I am yet to update it here